src/Library/demos/Status Page/main.blp

```diff
@@ -0,0 +1,58 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage content {
+
+  title: "No Results Found";
```

Comment:
All strings that are visible to users should be marked translatable, such as `_("No Results Found")`. Also see [GNOME HIG: Writing Style](https://developer.gnome.org/hig/guidelines/writing-style.html#capitalization) to be sure the correct capitalization is being used.